sqrl-tools/sqrl-packager/src/main/resources/visualize_dag.html

```diff
@@ -1,163 +1,30 @@
-                'curve-style': 'bezier',
-                'target-arrow-shape': 'triangle'
-            })
-            .selector('node[type="stream"]')
-            .css({
-                'shape': 'roundrectangle'
-            })
-            .selector('node[type="state"]')
-            .css({
-                'shape': 'rectangle'
-            })
-            .selector('node[type="relation"]')
-            .css({
-                'shape': 'cut-rectangle'
-            })
-            .selector('node[type="import"], node[type="export"]')
-            .css({
-                'font-weight': 'normal',
-                'shape': 'ellipse'
-            })
-            .selector('node[type="query"]')
-            .css({
-                'font-weight': 'normal',
-                'shape': 'tag'
-            }),
-
-        elements: {
-            nodes: cyNodes,
-            edges: cyEdges
-        },
-
-        layout: {
-            name: 'dagre'
-        }
-    });
-    cy.on('tap', 'node', function(evt){
-        let node = evt.target;
-        let type = node.data('type');
-        let content;
-        if (['stream','state','relation'].includes(type)) {
-            content = `
-                <p><strong>Name:</strong> ${node.data('name')}</p>
-                <p><strong>Type:</strong> ${node.data('type')}</p>
-                <p><strong>Stage:</strong> ${node.data('stage')}</p>
-                <p><strong>Schema:</strong></p>
-                <ul>
-                    ${node.data('schema').map(field => `<li>${field.name}: ${field.type}</li>`).join('')}
-                </ul>
-                <p><strong>Primary Key:</strong> ${node.data('primary_key').join(', ')}</p>
-                <p><strong>Timestamp:</strong> ${node.data('timestamp')}</p>
-                <p><strong>Id:</strong> ${node.data('id')}</p>
-                <p><strong>Plan:</strong></p>
-                <textarea style="width:100%;" rows="${Math.max(7,node.data('plan').split('\n').length+1)}" readonly>${node.data('plan')}</textarea>
-            `;
-        } else if (type === 'query') {
-            content = `
-                <p><strong>Type:</strong> ${node.data('type')}</p>
-                <p><strong>Id:</strong> ${node.data('name')}</p>
-                <p><strong>Plan:</strong></p>
-                <textarea style="width:100%;" rows="${node.data('plan').split('\n').length}" readonly>${node.data('plan')}</textarea>
-            `;
-        } else {
-            content = `
-                <p><strong>Name:</strong> ${node.data('name')}</p>
-                <p><strong>Type:</strong> ${node.data('type')}</p>
-                <p><strong>Id:</strong> ${node.data('id')}</p>
-            `;
-        }
-        document.getElementById('info-content').innerHTML = content;
-        document.getElementById('sideinfo').style.display = 'block';
-    });
-</script>
-</body>
-</html>
\ No newline at end of file
+      });
+
+      const edges = elements.flatMap((node) =>
```

Comment:
Can we move the transformation from elements to nodes and edges into the renderGraph method?